### ISSUE TITLE
when I replicate your c3d_train_ucf101 experement from scratch, the loss of the network never decrease.

### ISSUE THREAD

**Issue by haohao900618** [facebook/C3D#85]
Hi Du:
    I prepared data follwing the steps as https://github.com/facebook/C3D/issues/83. I run train_ucf101.sh without any modification, but I got the accuracy around 0 and the loss around 4.6 aways. I have send the output file to your email. Please remaind me Is there some steps I have overlooked. Thanks for your any help.

**Comment by haohao900618** [comment]
@TinkBell123 how did you solve the peoblem https://github.com/facebook/C3D/issues/49

**Comment by dutran** [comment]
@haohao900618 
Can you post the file list you prepared as well as the output log? Your file in the mail has a unknown format so I cannot read.

**Comment by haohao900618** [comment]
@dutran 
     Thanks for your attention. I have sent the two files to your email. please check your email. Thanks!

**Comment by dutran** [comment]
You can post your output log here. BTW, do you modify anything? What I saw is that you copy the the weights from the pre-trained model every 5K iterations,  what is the reason for that? And you also reduce the batch size to 10, you learning rate should be reduced with fine-tuning as well.

**Comment by haohao900618** [comment]
I have solved the problem by setting base_lr=0.001. Thank you a lot!
